wp-content/themes/inf_theme/admin/class-admin.php

```diff
@@ -37,14 +36,13 @@ public function enqueue_styles() {
    */
   public function enqueue_scripts() {
 
-    $main_script = General_Helper::get_manifest_assets_data( 'applicationAdmin.js' );
-    wp_register_script( static::THEME_NAME . '-scripts', $main_script, array(), static::THEME_VERSION, true );
+    wp_register_script( static::THEME_NAME . '-scripts', General_Helper::get_manifest_assets_data( 'applicationAdmin.js' ), array(), static::THEME_VERSION, true );
     wp_enqueue_script( static::THEME_NAME . '-scripts' );
 
   }
 
   /**
-   * Method that changes admin colors based on environment variable
+   * Method that changes admin colors based on environment variable. Must have INF_ENV global variable set.
```

Comment:
`Must have INF_ENV global variable set.` => `Must have INF_ENV global variable set to work.`